data/src/main/java/com/microsoft/azure/kusto/data/auth/endpoints/WellKnownKustoEndpointsData.java

```diff
@@ -44,8 +44,7 @@ private static WellKnownKustoEndpointsData readInstance() {
                 return objectMapper.readValue(resourceAsStream, WellKnownKustoEndpointsData.class);
             }
         } catch (Exception ex) {
-            ex.printStackTrace();
+            throw new RuntimeException("Failed to read WellKnownKustoEndpoints.json", ex);
```

Comment:
If you want you can annotate with @NotNull

Reply:
who? instance? 
May I not?

Reply:
the readInstance methods.

Just so callers can know to not expect null

you don't have to